### ISSUE TITLE
[Feature request] Add --version flag

### ISSUE THREAD

**Issue by twpayne** [doy/rbw#110]
It would be fantastic if `rbw` had a `--version` flag so that users can check what features are available.

Background:

rbw recently added support for raw JSON output in #97, and tagged version 1.6.0, which enabled chezmoi to support `rbw` as an interface to Bitwarden in #2836. Right now, chezmoi has no way of checking whether the `rbw` binary installed on the user's machine supports raw JSON output or not.

It would be great if the `rbw` binary could report its version.

**Comment by rjc** [comment]
Hmmm... this is strange - it *did* have it in version `1.4.3`:

```
$ rbw
rbw 1.4.3
Unofficial Bitwarden CLI

USAGE:
    rbw <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

SUBCOMMANDS:
    add                Add a new password to the database
    code               Display the authenticator code for a given entry
    config             Get or set configuration options
    edit               Modify an existing password
    gen-completions    Generate completion script for the given shell
    generate           Generate a new password [aliases: gen]
    get                Display the password for a given entry
    help               Prints this message or the help of the given subcommand(s)
    history            View the password history for a given entry
    list               List all entries in the local Bitwarden database [aliases: ls]
    lock               Lock the password database
    login              Log in to the Bitwarden server
    purge              Remove the local copy of the password database
    register           Register this device with the Bitwarden server
    remove             Remove a given entry [aliases: rm]
    stop-agent         Terminate the background agent
    sync               Update the local copy of the Bitwarden database
    unlock             Unlock the local Bitwarden database
    unlocked           Check if the local Bitwarden database is unlocked
```

but lost it somewhere between that and version `1.5.0`:

```
$ rbw
Unofficial Bitwarden CLI
                                                           
Usage: rbw <COMMAND>                                                                                                   
                                                           
Commands:                                         
  config           Get or set configuration options                                                                    
  register         Register this device with the Bitwarden server
  login            Log in to the Bitwarden server                                                                      
  unlock           Unlock the local Bitwarden database                                                                 
  unlocked         Check if the local Bitwarden database is unlocked
  sync             Update the local copy of the Bitwarden database                   
  list             List all entries in the local Bitwarden database [aliases: ls]
  get              Display the password for a given entry
  code             Display the authenticator code for a given entry  
  add              Add a new password to the database                                                                  
  generate         Generate a new password [aliases: gen]
  edit             Modify an existing password       
  remove           Remove a given entry [aliases: rm]                                                                  
  history          View the password history for a given entry
  lock             Lock the password database                                                                          
  purge            Remove the local copy of the password database
  stop-agent       Terminate the background agent
  gen-completions  Generate completion script for the given shell
  help             Print this message or the help of the given subcommand(s)

Options:
  -h, --help  Print help
```

**Comment by twpayne** [comment]
Ah, thanks for the tip. Some hasty use of `git bisect` indicates that commit 52efb7f9c09ea25136b0c2bc4c63455908ccf3bb is the one that removed the `--version` flag.

**Comment by doy** [comment]
fixed in 0509f30

**Comment by twpayne** [comment]
Thank you very much :)
